src/toolbox/system/exec.ts

```diff
@@ -5,7 +5,7 @@ import type { Device } from '../../types'
 
 function ensureAskPass(): void {
   const SUDO_ASKPASS = system.which('ssh-askpass')
-  if (SUDO_ASKPASS === undefined) {
+  if (SUDO_ASKPASS === null) {
```

Comment:
Looks like adding a check for `undefined` will appease the compiler:

```suggestion
  if (SUDO_ASKPASS === null || SUDO_ASKPASS === undefined) {
```

Reply:
Thanks for that. I just tested it in my environment and it works as describe!